test/openAPI/features/data_list.feature

```diff
@@ -1,17 +1,45 @@
-Feature: API endpoint that allows users to search multiple records in Digital Registries database.
-  Request endpoint: GET /data/{registryname}/{versionnumber}/
+@method=GET @endpoint=/data/{registryname}/{versionnumber}/
+Feature: API endpoint that allows to search multiple records in the database.
 
-  Scenario: The user gets a list of all searched records in the Digital Registries database
-    Given The user wants to search for a specific value and the searched value exists in multiple records in the database
-    When The user sends a valid request to search the database
-    Then The user receives a list of all records that contain the searched value
+  @smoke
+  Scenario: Successfully obtains database users information with database schema smoke type test
+    Given get the database users information of Digital Registries
```

Comment:
given is a kind of precondition. we describe here who is going to do what. 
For example 
user wants to get the database users information